src/parser/Parser.ts

```diff
@@ -1321,11 +1320,12 @@ export class Parser {
     /**
      * Get an expression with identifiers separated by periods. Useful for namespaces and class extends
      */
-    private getNamespacedVariableNameExpression() {
+    private identifyingExpression(allowedTokenKinds?: TokenKind[]): DottedGetExpression | VariableExpression {
```

Comment:
Makes this function work for finding any VariableExpression or DottedGetExpression - more reusable